### ISSUE TITLE
Can you finetune the pre-trained algoritms in scispacy to add more labels?

### ISSUE THREAD

**Issue by kmnhops** [allenai/scispacy#467]
I am new to spacy. I wanted to know if we can fine-tune scispacy models like we can do with spacy model by just making a spacy config file, after we convert the data to used for fine-tuning to spacy format.

**Comment by dakinggg** [comment]
Hi @kmnhops, yes this should work! scispacy models are just scispacy models :)
